packages/google_maps_routes_api/.gitignore

```diff
@@ -0,0 +1,30 @@
+# Miscellaneous
```

Comment:
Check if these files are needed. At least with plugins there is a .gitignore file in the root of whole packages folder that is used.

Reply:
Each package seems to have it's own .gitignore as well, but this could be cleaned up a bit.